src/AnswersHeadlessProvider.tsx

```diff
@@ -2,7 +2,7 @@ import { ReactChild, ReactChildren } from 'react';
 import { provideAnswersHeadless, AnswersHeadless, HeadlessConfig } from '@yext/answers-headless';
 import { AnswersHeadlessContext } from './AnswersHeadlessContext';
 import acquireSessionId from './utils/acquireSessionId';
-import packageJson from '../package.json';
+import packageJson from '../package.json' assert { type: 'json' };
```

Comment:
didn't know about [import assertions](https://v8.dev/features/import-assertions)!
do we also need to update our other repos that import json files?

Reply:
Tom will talk to Max about them since older versions of node blow up when they encounter them. Support was added in node 17.1 and was made required in node 17.5 according to a [stackoverflow post](https://stackoverflow.com/questions/70106880/err-import-assertion-type-missing-for-import-of-json-file). It was also added to 16.14 however Tom wanted us to simply require node 17.1 for now and publish an alpha for testing purposes

Reply:
The import assertion is causing me some issues since Vite doesn't support it, so we can leave it out for now